generate.js

```diff
@@ -78,8 +82,14 @@ const template = `
         </div>
       {{/orgs}}
     </div>
+    <script>
+      ${script}
```

Comment:
`<script src="static/js/app.js"></script>`

Reply:
the mustache template doesn't work